proto/resources.proto

```diff
@@ -0,0 +1,75 @@
+syntax = "proto3";
+
+package rs.tokio.console.resources;
```

Comment:
take it or leave it: IMO, the `resources` and `resource_ops` protos are simple enough I would probably just combine them into one file...but it's up to you.

Reply:
Makes sense, will combine